arches_references/management/commands/controlled_lists.py

```diff
@@ -169,15 +171,21 @@ def migrate_collections_to_controlled_lists(
             result = cursor.fetchone()
             self.stdout.write(result[0])
 
-    def migrate_concept_nodes_to_reference_datatype(self, graph_id):
+    def migrate_concept_nodes_to_reference_datatype(self, graph):
```

Comment:
We need to make sure `--graph` exists and is not empty, because if it comes in missing, it converts to slug=None, which can match multiple graphs and cause `get()` to throw:
```py
  File "/Users/jwalls/prj/arches-references/arches_references/management/commands/controlled_lists.py", line 181, in migrate_concept_nodes_to_reference_datatype
    source_graph = GraphModel.objects.get(query)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/release/lib/python3.12/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/release/lib/python3.12/site-packages/django/db/models/query.py", line 652, in get
    raise self.model.MultipleObjectsReturned(
arches.app.models.models.GraphModel.MultipleObjectsReturned: get() returned more than one GraphModel -- it returned 3!
```